src/connection/EventListener.ts

```diff
@@ -0,0 +1,24 @@
+export interface EventListener {
+  onEvent(eventType: string, eventProperties?: any): void
+}
+
+export const EVENTS = {
+  REQUEST_CHILD_ETHEREUM_ACCOUNTS_START: "cbwsdk.request_child_ethereum_accounts.start",
```

Comment:
will changing event names break tracking continuity? since these are only used internally it's more important that we don't disrupt any analytics dashboards/alerts

Reply:
yeah, i made the change on our extension as well to match these, but i might just keep them unchanged. 
![image](https://user-images.githubusercontent.com/966662/156068264-20f544a0-2002-47f6-b0fa-76bb39ab949f.png)

Reply:
reverted it on extension